prusa_slicer_post_processing_script.py

```diff
@@ -37,6 +37,7 @@
 import platform
 #from hilbertcurve.hilbertcurve import HilbertCurve
 from hilbert import decode, encode
+import argparse
```

Comment:
Just nitpicking here: I think this would make more sense so import on line 29, after `os`.